lib/device.js

```diff
@@ -374,7 +367,7 @@ const async = require('async'),
     };
 
     function makeSession(options, next){
-        log.info("device#makeSession");
+        log.info("device#makeSession()", (options.session ? "already session connected" : "need to make new session"));
```

Comment:
I don't think it's appropriate to do this.
It's cumbersome to log if you actually hit the logic, but it's safe.

I think it would be better to change as below
#370, log.info("device#makeSession()"); 
after #372, log.info(make new session);
after #375, log.info(already exist session);

Reply:
I replaced it to log.info(make new session) and log.info(already exist session) in if-else block. 
Done.